### PR TITLE
PSD-6: More account renaming

### PR DESCRIPTION
**Pull request by broder**
And other tech debt.

## Description
Probably worth going commit by commit

## Checklist:

- [x] Automated checks are passing locally.
- [x] CHANGELOG updated if change is worth telling users about.

### CODE REVIEW

File: psd-web/Gemfile.lock [hunk 1]
```diff
@@ -455,7 +455,7 @@ DEPENDENCIES
   wicked
 
 RUBY VERSION
-   ruby 2.5.3p105
+   ruby 2.5.5p157
 
 BUNDLED WITH
-   1.17.2
+   1.17.3
```
Comment (tom-bell-softwire) [inline psd-web/Gemfile.lock]: Could we use v2.0.1 across the board?

Comment (broder) [reply psd-web/Gemfile.lock]: Ah, I thought I had kept it as 1.17.3 everywhere but it looks like I missed one.

The `ruby:2.5.5` docker image comes with 1.17.3 by default so it's probably easiest to just use that. The 2.0.1 crept in when I was using my system ruby to update the `Gemfile.lock`s

Comment (broder) [reply psd-web/Gemfile.lock]: Ah, I thought I had kept it as 1.17.3 everywhere but it looks like I missed one.

The `ruby:2.5.5` docker image comes with 1.17.3 by default so it's probably easiest to just use that. The 2.0.1 crept in when I was using my system ruby to update the `Gemfile.lock`s

File: cosmetics-worker/manifest.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 applications:
 - name: cosmetics-worker
-  memory: 512M
+  memory: 1G
```
Comment (jasiekmiko) [inline cosmetics-worker/manifest.yml]: was there a particular issue this is trying to solve?

Comment (broder) [reply cosmetics-worker/manifest.yml]: It was a speculative fix for https://sentry.io/organizations/beis/issues/978206675/?project=1329381&query=is%3Aunresolved&statsPeriod=14d

**Pull request merged**